### PR TITLE
[FEATURE typography] Add `style` prop to interface

### PR DESCRIPTION
**Pull request by damassi**
Adds ability to override typography props via `style`.

### CODE REVIEW

File: src/elements/Typography.tsx [hunk 1]
```diff
@@ -68,6 +68,7 @@ export interface TextProps
   fontFamily?: string
   fontSize: number
   lineHeight: number
+  style: CSSProperties
```
Comment (alloy) [inline src/elements/Typography.tsx]: This is a required prop, is that intentional? The description of the PR makes it sound optional.

Comment (damassi) [reply src/elements/Typography.tsx]: Ah, good catch -- fixed 👍

**Comment by alloy** [comment]
Ace 👍 Merge on green.

**Comment by damassi** [comment]
Danger seems to be hanging -- going to go ahead and merge since other status's are 🍏

**Pull request merged**